### DISCUSSION TITLE
PNP Search Filter Webpart Help for Multiple Choice Column

### DISCUSSION THREAD

**Discussion by huetvictoria** [microsoft-search/pnp-modern-search#2513] [Q&A] [answered]
Hi, I am trying to add a search filter for a Site Column that is a Choice column that allows multiple values. The values are displaying as the multiple values, instead of a single option to allow users to search by a single value. Please advise how to allow a document to have multiple values but have the Search Filter simply show the single value. It is also adding random # symbols to some of the values. Here is a snapshot of how it is displaying:
![image](https://user-images.githubusercontent.com/67760631/199972785-f6682bc8-0a9b-4d9f-a3af-92e35511d3bf.png)

**Comment by wobba** [comment]
No easy way as this is how choice columns are indexed.

**Reply by huetvictoria** [reply]
But is there a way?

**Reply by wobba** [reply]
If you develop a custom refinement component which split the string and rework it. Which would be valuable to have, but no one has addressed this so far.

**Reply by Spenserad** [reply, marked as answer]
I was intermittently running into this issue too, the solution seemed to be adjusting the crawled property mapping to the Refinable managed property. For example, I had my RefinableString00 managed property mapped to the crawled property "ows_q_CHCM_ChoiceField" and that resulted in the behavior you're describing, however, when I changed the mapping over to "ows_ChoiceField" (because the system creates both) the refiner worked as expected, parsing the values apart for individual selection instead of the full string.

**Reply by wobba** [reply]
@Spenserad nice to know :)
